src/utils/helpers.js

```diff
@@ -0,0 +1,33 @@
+import haversine from 'haversine'
+
+export const distanceFilter = 50
+
+export function getClosestParksByID(type, id, currentLocation, parks) {
```

Comment:
nitpick: `getClosestParksByType` might be more accurate

Reply:
Most accurately would be `getClosestParksByTypeAndID` 😉